pkg/middlewares/metrics/metrics.go

```diff
@@ -131,6 +144,88 @@ func getMethod(r *http.Request) string {
 	return r.Method
 }
 
+func getRequestTLSVersion(req *http.Request) string {
+	var tlsVersion string
+
+	switch req.TLS.Version {
+	case tls.VersionTLS10:
+		tlsVersion = "tls-1.0"
```

Comment:
This really isn't in a format that is commonly used, could these labels be updated to something more standard?

Reply:
At first I only put the version number as value and then when I noticed there still was SSL v3 in go I added the protocol name.

I don't really care, I just don't really like spaces in label values.